avocado/plugins/resolvers.py

```diff
@@ -69,11 +69,12 @@ def resolve(reference):
                 mod = mod[:-3]
             mod = mod.replace(os.path.sep, ".")
             for klass, meths in class_methods.items():
-                for (meth, tags) in meths:
+                for (meth, tags, reqs) in meths:
                     uri = '%s.%s.%s' % (mod, klass, meth)
                     runnables.append(Runnable('python-unittest',
                                               uri=uri,
-                                              tags=tags))
+                                              tags=tags,
+                                              requirements=reqs))
```

Comment:
@clebergnu the only thing that changed from previous merged/reverted PR (#3725) is this block.